docs/Schema.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE card(
```

Comment:
For documentation purpose, maybe also add comments above each "CREATE TABLE" statement on what this table contains, and how it relates to other tables.